config/host-list.json

```diff
@@ -0,0 +1,12 @@
+{
+"node_label": "label",
+"executors": "",
+"template_detail": {
+"guest_username": "root",
+"guest_password": "passwd",
+"guest_user_credentials_id": "xxxxxxxx"

```

Comment:
Why are these settings specific to the guest user? Can't it be done by any regular user?